src/datachain/progress.py

```diff
@@ -1,138 +1,16 @@
 """Manages progress bars."""
 
 import logging
-import sys
 from threading import RLock
-from typing import Any, ClassVar
 
 from fsspec import Callback
 from fsspec.callbacks import TqdmCallback
-from tqdm import tqdm
-
-from datachain.utils import env2bool
+from tqdm.auto import tqdm
 
 logger = logging.getLogger(__name__)
 tqdm.set_lock(RLock())
```

Comment:
Let's remove this too when we can.

Maybe too much detail, but [`tqdm` does locking by default which is multiprocess-safe](https://github.com/tqdm/tqdm/blob/0ed5d7f18fa3153834cbac0aa57e8092b217cc16/tqdm/std.py#L76). But since we overwrite this, this is only thread-safe.

[`dvc-data` does reset it, so we'll be overriding it anyway as we are importing it someplace](https://github.com/iterative/dvc-data/blob/8e28434fd4f07f948f018350abe86928094312ea/src/dvc_data/callbacks.py#L12). But I hope we can remove it someday there. I have no idea why this was done in `dvc`.

Reply:
(I have a suspicion that this is what causes progress to jump in dvc)

Reply:
yep, makes sense

Reply:
`dvc-data` fixed in https://github.com/iterative/dvc-data/pull/592.